tools/python

```diff
@@ -185,6 +185,27 @@ async function main() {
         process.exit(code);
     };
 
+    py.runPython(
+        `
+        from pyodide._package_loader import SITE_PACKAGES, should_load_dynlib
+        from pyodide.ffi import to_js
+        dynlibs_to_load = to_js([
+            str(path) for path in SITE_PACKAGES.glob("**/*.so")
```

Comment:
Could you use a same regex used in https://github.com/pyodide/pyodide/pull/3154, just in case?